.gitignore

```diff
@@ -5,4 +5,6 @@
 example/opencv_demo
 example/apriltag_demo
 build/
-
+build-debug/
+build-release/
+.idea/
```

Comment:
Those are IDE specific. I am in favour of not adding all IDE-specific files here, but only keeping those that are produced by the documented make or make procedure.

Reply:
the flavor builds are not IDE specific, are you ok those being simplified to build-*/?
I removed .idea - also I just learned about `git config --global core.excludesfile ~/.gitignore_global`